packages/core/src/layout/InfoCard/BottomLink.js

```diff
@@ -17,9 +17,24 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Link } from '@material-ui/core';
-import { Divider, ListItemText } from '@material-ui/core';
+import { Divider, ListItemText, withStyles } from '@material-ui/core';
 import { ListItem, ListItemIcon } from '@material-ui/core';
 import ArrowIcon from '@material-ui/icons/ArrowForward';
+import grey from '@material-ui/core/colors/grey';
+
+const LinkBottomText = withStyles({
+  primary: {
+    fontWeight: '700',
+    color: grey[900],
+  },
+})(ListItemText);
+
+const ListItemBottom = withStyles({
+  root: {
+    maxWidth: 'fit-content',
+    padding: '16px 16px 16px 20px',
```

Comment:
For all spacing, make use of the theme; in this instance, `theme.spacing(2, 2, 2, 2.5)` (if we are using fractional spacing @Katz)

Reply:
awesome! Thanks, Fredrik! :smile: